examples/byoc-with-sagemaker-containers/scikit_bring_your_own_with_sagemaker_containers.ipynb

```diff
@@ -0,0 +1,760 @@
+{
+ "cells": [
```

Comment:
Are you planning on replacing the original version of this notebook?

Reply:
Yes, I wanted a smaller example here as well.